src/main.ts

```diff
@@ -3,21 +3,12 @@ import * as exec from '@actions/exec'
 import * as github from '@actions/github'
 import * as tc from '@actions/tool-cache'
 import {chmodSync} from 'fs'
-import {cliInfo} from './mapiapi'
 import slugify from 'slugify'
 
 // Return local path to donwloaded or cached CLI
 async function mapiCLI(): Promise<string> {
   // Get latest version from API
-  let cliVersion = 'latest'
-  try {
-    cliVersion = (await cliInfo()).latest_version
```

Comment:
Even though we're fetching the `latest`, I think we still need to grab the version from `https://app.mayhem.security/cli/mapi/${os}/latest/${bin}.meta`, because of the tool cache.

Reply:
Done in c272f9674c7b14d3766725835d2f08e04924fc28.